### DISCUSSION TITLE
Kernel selection fails

### DISCUSSION THREAD

**Discussion by Rasb74** [rern/rAudio#151] [Fixed Issues]
Hi Rern, 
Trying to test different kernels, unfortunately the system failed. 
![Capture d’écran 2021-01-30 à 23 27 05](https://user-images.githubusercontent.com/77485841/106369607-28084c80-6353-11eb-93f1-0ae0b2a99c82.jpg)

Congrats for your great job !

**Comment by rern** [comment]
Please try updated image files which have the latest kernel and packages. Let me know whether it's still OK with your RPi 4.

**Comment by Rasb74** [comment]
Hello !
Looks working well now. 
It seems the kernel Orion V4 give the best quality on my system.
I must go deeper in this investigation, I’ll let you know, ay it’s interesting for you isn’t it?
Cheers
Pascal
